src/Controllers/ItemsController.cs

```diff
@@ -0,0 +1,78 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Options;
+using Phytime.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.ServiceModel.Syndication;
+using System.Xml;
+
+namespace Phytime.Controllers
+{
+    [ApiController]
+    [Route("api/[controller]")]
+    public class ItemsController : Controller
+    {
+        private readonly IRepository<Feed, User> _feedRepository;
+
+        public ItemsController(IOptions<ConnectionStringsOptions> options, IRepository<Feed, User> repository = null)
+        {
+            _feedRepository = repository ?? new FeedRepository(options.Value.DefaultConnection);
+        }
+
+        [HttpGet("{id:int}")]
+        public IEnumerable<Item> Get(int id)
+        {
+            if (id < 1)
```

Comment:
replace magic number with const variable, describe why you need that number in name

if(id < FirstValidId)